README.md

```diff
@@ -62,6 +62,7 @@ Here is a list of variables that you can use to customize your newly copied `.en
 
 | **Environment variable** | **Description** | **Notes** |
 |:---:|---|---|
+| `USER_AGENT` | Custom User-Agent header for HTTP requests; optional | 
```

Comment:
Do you mind putting this in order?

Same with example?

Reply:
Done.  I didn't realize they were alphabetized.